ros/lib/models.py

```diff
@@ -28,6 +28,15 @@ def display_performance_score(self):
 
         return display_performance_score
 
+    @property
+    def idling_time(self):
+        return (self.performance_record['kernel.all.cpu.idle']
+                / self.performance_record['total_cpus']) * 100
+
+    @property
+    def io_wait(self):
+        return self.performance_record['kernel.all.cpu.wait.total'] * 100
```

Comment:
Here we are calculating idling_time and io_wait in `percentage` but in the mockups, they are shown in `mins`.

Reply:
Good catch, fixed! :+1: